qiskit/primitives/base/base_estimator.py

```diff
@@ -255,19 +255,15 @@ def _run(
     ################################################################################
     @staticmethod
     def _validate_observables(
-        observables: Sequence[BaseOperator | PauliSumOp] | BaseOperator | PauliSumOp,
-    ) -> tuple[BaseOperator | PauliSumOp, ...]:
-        if isinstance(observables, (BaseOperator, PauliSumOp)):
+        observables: Sequence[BaseOperator | PauliSumOp | str] | BaseOperator | PauliSumOp | str,
+    ) -> tuple[SparsePauliOp, ...]:
+        if not isinstance(observables, Sequence):
```

Comment:
You may simplify the code by combining the condition before return.
```suggestion
        if isinstance(observables, str) or not isinstance(observables, Sequence):
```

Reply:
Done! Thanks a lot! 🙂